app-base/src/main/java/com/tapadoo/alerter_demo/DemoActivity.java

```diff
@@ -36,98 +36,77 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onClick(View view) {
-        switch (view.getId()) {
-            case R.id.btnAlertColoured: {
-                showAlertColoured();
-                break;
-            }
-            case R.id.btnAlertCustomIcon: {
-                showAlertWithIcon();
-                break;
-            }
-            case R.id.btnAlertTextOnly: {
-                showAlertTextOnly();
-                break;
-            }
-            case R.id.btnAlertOnClick: {
-                showAlertWithOnClick();
-                break;
-            }
-            case R.id.btnAlertVerbose: {
-                showAlertVerbose();
-                break;
-            }
-            case R.id.btnAlertCallback: {
-                showAlertCallbacks();
-                break;
-            }
-            case R.id.btnAlertInfiniteDuration: {
-                showAlertInfiniteDuration();
-                break;
-            }
-            case R.id.btnAlertWithProgress: {
-                showAlertWithProgress();
-                break;
-            }
-            case R.id.btnAlertWithCustomFont: {
-                showAlertWithCustomFont();
-                break;
-            }
-            case R.id.btnAlertSwipeToDismissEnabled: {
-                showAlertSwipeToDismissEnabled();
-                break;
-            }
-            default: {
-                showAlertDefault();
-            }
+        int i = view.getId();
```

Comment:
I'm curious, why this changed to an if/else?

Reply:
Turns out you cannot switch on resources id's in com.android.feature modules - this seems like a bug to me. 😠